billing/main.py

```diff
@@ -0,0 +1,63 @@
+import base64
+import json
+import os
+from googleapiclient import discovery
```

Comment:
Can the GAPIC libraries be used instead? https://github.com/googleapis/python-billing and  https://github.com/googleapis/python-billingbudgets? (not sure which is applicable here).

Reply:
I'm not familiar with the GAPIC libraries. What is the benefit or reason for using them versus the one currently being used?

Reply:
[This page](https://cloud.google.com/apis/docs/client-libraries-explained) explains the main differences. (GAPIC == Google Cloud Client Library)